uber/server.py

```diff
@@ -28,13 +28,13 @@ def magfest_js(self):
         We also need those apps to be able to make HTTP requests with CSRF tokens, so we set that default.
         """
         cherrypy.response.headers['Content-Type'] = 'text/javascript'
```

Comment:
This is the place where we actually render all of our config options to JSON and send it to the browser.  This has been broken since the ``c`` refactor but hasn't come up until now that we're about to launch shift signups.

This should be fairly intuitive to anyone who knows Angular and fairly incomprehensible to anyone who doesn't :)